dom-manipulations-homework/index.ts

```diff
@@ -0,0 +1,51 @@
+// Add event listener when button with id "add-border-btn" is clicked
+// and add border style "2px solid red" for section with class name "content"
+
+const addBorderBtn = document.getElementById("add-border-btn");
+const contentSection = document.querySelector(".content") as HTMLElement;
+
+if (addBorderBtn && contentSection) {
+  addBorderBtn.addEventListener("click", () => {
+    contentSection.style.border = "2px solid red";
+  });
+}
+
+
+
+// Add event listener when button with id "change-text-btn" is clicked
+// and add color style "blue" for section with class name "content"
+
+const changeTextBtn = document.getElementById("change-text-btn");
+
+if (changeTextBtn && contentSection) {
+  changeTextBtn.addEventListener("click", () => {
+    contentSection.style.color = "blue";
+  });
+}
+
+
+// Add event listener when button with id "change-text-btn" is clicked
+// and add backgroundColor style "gray" for section with class name "content"
+
+const changeBackgroundBtn = document.getElementById("change-background-btn");
+
+if (changeBackgroundBtn && contentSection) {
```

Comment:
Change background functionality don't work. Select correct button for changing background color "toggle-bg-btn" and then attach event listener for changing the background to section with id content